analysis/report_templates/default.html

```diff
@@ -232,17 +232,51 @@ <h5 class="center-align">Mann-Whitney U test</h4>
                 </li>
             </ul>
 
-            {% if coverage_report %}
+            {% if detailed_coverage_report %}
+            <ul class="collapsible">
+                <li>
+                    <div class="collapsible-header">
+                        Differential matrix
```

Comment:
Consider: Differential matrix -> Unique coverage plots

(it doesn't only contain the matrix)